Core/src/org/sleuthkit/autopsy/communications/FiltersPanel.java

```diff
@@ -196,8 +197,8 @@ private void updateAccountTypeFilter() {
         Account.Type.PREDEFINED_ACCOUNT_TYPES.forEach(type -> {
             if (type.equals(Account.Type.CREDIT_CARD)) {
                 //don't show a check box for credit cards
-            } else if (type.equals(Account.Type.DEVICE)) {
-                //don't show a check box fro device
+//            } else if (type.equals(Account.Type.DEVICE)) {
+//                //don't show a check box fro device
```

Comment:
Is this just a temporary removal? If not, I think it would be better to delete rather than comment out.

Reply:
removed